public/app/plugins/datasource/mssql/partials/config.html

```diff
@@ -0,0 +1,40 @@
+
+<h3 class="page-heading">MSSQL Connection</h3>
+
+<div class="gf-form-group">
+	<div class="gf-form max-width-30">
+		<span class="gf-form-label width-7">Host</span>
+		<input type="text" class="gf-form-input" ng-model='ctrl.current.url' placeholder="localhost:1433" bs-typeahead="{{['localhost:1433', '192.168.0.101:1433']}}" required></input>
+	</div>
+
+	<div class="gf-form max-width-30">
+		<span class="gf-form-label width-7">Database</span>
+		<input type="text" class="gf-form-input" ng-model='ctrl.current.database' placeholder="database name" required></input>
+	</div>
+
+	<div class="gf-form-inline">
+		<div class="gf-form max-width-15">
+			<span class="gf-form-label width-7">User</span>
+			<input type="text" class="gf-form-input" ng-model='ctrl.current.user' placeholder="user"></input>
+		</div>
+		<div class="gf-form max-width-15">
+			<span class="gf-form-label width-7">Password</span>
+			<input type="password" class="gf-form-input" ng-model='ctrl.current.password' placeholder="password"></input>
```

Comment:
Should use secureJsonData to encrypt the password like the Postgres data source does:

https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/postgres/partials/config.html#L24-L28

and decrypt it on the backend:

https://github.com/grafana/grafana/blob/master/pkg/tsdb/postgres/postgres.go#L49

Reply:
Done!